### ISSUE TITLE
Jaws in IE11 and VideoJS Volume not changing with keyboard

### ISSUE THREAD

**Issue by Chris-Developer** [videojs/video.js#4601]
## Description
Using JAWS 18.0.2118 in IE11, VideoJS will not change the volume using the arrow keys. Up, Down, Left and Right do not work. This affects both video and audio players.

## Steps to reproduce
1. Navigate to http://videojs.com/
2. With JAWS launched, use tab to navigate to the volume slider.
3. Try to change the volume slider with the arrow keys.

### Expected
Volume is able to be increased/decreased with the arrow keys, like it does when JAWS is not active.

### Actual
The volume does not change.

### Error output
None

## Additional Information
Please include any additional information necessary here. Including the following:
6.2.7 videojs - recreatable on videojs site
IE11 - 11.0.9600.18763
JAWS 18.0.2118
Windows 8.1 Enterprise

**Comment by OwenEdwards** [comment]
We can check this somewhere else, but FYI http://videojs.com/ uses the latest version of video.js 5.x, not 6.x. @gkatsev, this may need clarification on the http://videojs.com/ site.

**Comment by Chris-Developer** [comment]
@OwenEdwards,

Thanks for following up. My mistake on not being clearer. We are using v5.8.6 and given your comment, I have tested this on v6.2.7 and the issue is resolved. Seems like this was fixed when the volume bar was split out from the mute button.

This would indicate why it was broken on videojs.com. It would definitely help making it clearer which version videojs.com uses, my assumption was it's always the latest. Good to know that it isn't.

I'll need to assess the feasibility of upgrading to v6.2.7 and if it causes any regressions in our product.

Thanks for the clarification.
